draft-ietf-ppm-dap-taskprov.md

```diff
@@ -209,16 +209,26 @@ struct {
 
     /* Determines the VDAF type and its config parameters. */
     opaque vdaf_config<1..2^16-1>;
+
+    /* Taskbind Extensions. */
+    Extension extensions<0..2^16-1>;
 } TaskConfig;
 ~~~
 
 The purpose of `TaskConfig` is to define all parameters that are necessary for
-configuring each party. It includes all the fields to be associated with a
-task. It also includes an opaque `task_info` field that is specific to a
-deployment. For example, this can be a string describing the purpose of this
-task. It does not include cryptographic assets shared by only a subset of the
-parties, including the secret VDAF verification key {{!VDAF}} or public HPKE
-configurations {{!RFC9180}}.
+configuring each party. It includes all parameters listed in {{Section 4.3 of
+!DAP}} as well as two additional fields:
+
+* `task_info` is an opaque field whose contents are specific to the deployment.
+  For example, this might be a human-readable string describing the purpose of
+  this task.
```

Comment:
This smells like an extension to me.

Reply:
I'm not sure it should be if it's mandatory to implement. In any case, I think we should discuss removing the task_info field in a separate issue.